### DISCUSSION TITLE
How to gracefully shut down the Stash process?

### DISCUSSION THREAD

**Discussion by alephalpha0** [stashapp/stash#2225] [Q&A] [answered]
So, I'm trying to reboot my Stash process and I'm at a loss of what SIG to send it... Any help?

**Comment by kermieisinthehouse** [comment, marked as answer]
Ctrl-C / SIGINT is fine, or SIGTERM.

**Comment by alephalpha0** [comment]
Thank you! I kinda figured, but didn't want to assume and start degrading my install right from the get go. 

-- 
Mors certa; vita incerta.

Trust no one. //



Jan 9, 2022, 05:14 by ***@***.***:
